lib/api/azure/base.ex

```diff
@@ -25,8 +25,22 @@ defmodule FileStorageApi.API.Azure.Base do
     end
   end
 
-  def container(container_name, connection_name) do
-    Container.new(storage(connection_name), container_name)
+  def storage(connection) do
+    config = Map.get(connection, :config, [])
+
+    if Map.get(config, :development) do
```

Comment:
the default is a list and you do Map.get here? Maybe you can use the Access module so it works for maps and keylists

Reply:
Changed all the Map.get from dynamic config to Access.get